demo_data/users.json

```diff
@@ -59,6 +59,36 @@
       "activity_streams_email_notifications": false,
       "state": "active",
       "fullname": "Celine Close"
+    },
+    {
+      "email": "anna2@fjelltopp",
```

Comment:
Instead of Anna2 please maybe use:
south_pole_member@fjelltopp.org
display_name: Member SouthPole
That would be more descriptive and help in local dev env.

Reply:
The same comment goes to other users as well.

Reply:
> Why do we need this 3rd demo organisation? What additional value does it bring which can't be achieved with already existing Fjelltopp and North Pole?

I needed it to do UI testing with my dataset transfer feature: org_1 wants to transfer a dataset to org_2 however we need an org_3 to make sure that org cannot interfere with that transfer.

Reply:
Ok, I get it. That's fine.
Could you just rename the users as I've suggested. I was also thinking we could create 3 users being members of two organisations at once (notrth pole and south pole). That would also be handy in local dev.

Reply:
Added `santa_claus` who's a member of both North and South poles ✅